content/en/docs/concepts/configuration/scheduling-framework.md

```diff
@@ -15,11 +15,10 @@ that makes scheduler customizations easy. It adds a new set of "plugin" APIs to
 the existing scheduler. Plugins are compiled into the scheduler. The APIs
 allow most scheduling features to be implemented as plugins, while keeping the
 scheduling "core" simple and maintainable. Refer to the [design proposal of the
-scheduling framework][kep] for more technical information on the design of the
+scheduling framework](https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20180409-scheduling-framework.md) 
+for more technical information on the design of the
 framework.
 
-[kep]: (https://github.com/kubernetes/enhancements/blob/master/keps/sig-scheduling/20180409-scheduling-framework.md)
```

Comment:
Technically, all you needed to do to fix this was remove the parentheses on this line around the link :)

https://daringfireball.net/projects/markdown/syntax#link

Reply:
thanks, updated